### PR TITLE
Added UPchieve logo to login/register

### PR DESCRIPTION
**Pull request by zuoyuanh**

### CODE REVIEW

**Comment by zuoyuanh** [comment]
So I want to place the logo in the center of the empty space above the white form box. I tried to use div but the logo will push the form off. I tried to fix the position of the logo to the top of the page, but if user resize the screen or the screen is very large, the logo will not be centered vertically.

**Comment by lualparedes** [comment]
@zuoyuanh try using Flexbox, HTML tables should be avoided unless you're actually working with tabular data (it's a matter of semantics).

Here are some resources that might help:
- https://www.lifewire.com/tables-for-tabular-data-3469858
- https://philipwalton.github.io/solved-by-flexbox/demos/vertical-centering/
- https://css-tricks.com/snippets/css/a-guide-to-flexbox/

**Comment by lualparedes** [comment]
@zuoyuanh @zuoyuanh97  Do your latest edits address all of the changes requested?

**Comment by alymurray** [comment]
Logo scales very strangely as you change window's size. I'd instead like the size of the logo to stay fixed in relation to the size of the white box.

image


Here's how it looks in the wireframes for reference

image

File: src/components/LoginForm.vue [hunk 1]
```diff
@@ -211,4 +220,27 @@ export default {
...
+  .main-page {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    height: 100%;
+  }
+  
+  .logo-img {
+    width: 35%;
```
Comment (stackptr) [inline src/components/LoginForm.vue]: Set a `max-width` here to avoid the scaling issues that @alymurray is noticing

**Comment by jmromer** [comment]
Rebased with fixes applied. The login form is still somewhat off-center — creating a ticket for that.


#### After

register
login

**Comment by jmromer** [comment]
Tested on staging

Screen Shot 2019-03-25 at 1 19 52 PM

**Pull request merged**